src/sentry/search/events/fields.py

```diff
@@ -2358,23 +2358,27 @@ def resolve_orderby(self, orderby: Optional[Union[List[str], str]]) -> List[Orde
 
         for orderby in orderby_columns:
             bare_orderby = orderby.lstrip("-")
-            resolved_orderby = self.resolve_field(bare_orderby)
+            try:
+                resolved_orderby = self.resolve_field(bare_orderby)
+            except NotImplementedError:
+                resolved_orderby = None
```

Comment:
Is this cause of doing `resolve_field` on functions raises notImplementedError? could we just do `else` on L2370 so we only run resolve_field on fields?

Reply:
resolve_field on function columns works fine (for example `failure_count()` will work) but function aliases doesn't (for ex. `failure_count`). yeah I change the order of resolution from column > field/function to field/function > column. I can't think of any side effects of that, I think it should be fine

Reply:
Ah ok yeah i think this is fine, but actually now that I see that can we do `resolved_orderby.alias` instead of having to run `get_function_alias` directly?